docs/README.android

```diff
@@ -182,12 +157,13 @@ probably just means a debug key already existed, no cause for alarm.
       android -storepass android -keyalg RSA -keysize 2048 -validity 10000
 
 -----------------------------------------------------------------------------
-4. Getting the source code
+4. Getting the source code for Krypton
+   We are getting 17RC1
```

Comment:
This is wrong because this readme is only added to master branch and it should not contain specific branch instruction besides "master". Yes you can add some comment or note to say they can pull different branch